<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
font: 600 11pt &quot;Segoe UI Variable Display Semib&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="background_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>900</width>
      <height>506</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Build2/img/login.jpg</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="signIn_Button">
    <property name="geometry">
     <rect>
      <x>252</x>
      <y>378</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: transparent;
border: 2px solid transparent</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="create_Button">
    <property name="geometry">
     <rect>
      <x>532</x>
      <y>388</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: transparent;
border: 2px solid transparent</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name_lineEdit">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>230</y>
      <width>431</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:25px solid transparent;
background-color: transparent;

font: 600 11pt &quot;Segoe UI Variable Display Semib&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="pswd_lineEdit">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>310</y>
      <width>441</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:25px solid transparent;
background-color: transparent;

font: 600 11pt &quot;Segoe UI Variable Display Semib&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="error_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>961</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 600 14pt &quot;Myanmar Text&quot;;
color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="lookpsw_Button">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>300</y>
      <width>75</width>
      <height>65</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>75</width>
      <height>65</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px solid  rgb(24, 205, 255);
border-radius: 30px;
background-color: transfert;
font: 600 8pt &quot;Segoe UI Variable Display Semib&quot;;</string>
    </property>
    <property name="text">
     <string>посмотреть</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
